src/site/includes/header.html

```diff
@@ -80,8 +80,8 @@
 
   {% include "src/site/components/fullwidth_banner_alerts.drupal.liquid" %}
 
-  {% if path == '' or path == '/' %}
-  <div data-widget-type="homepage-banner"></div>
+  {% if homepage_banner %}
+    {% include "src/site/includes/homepage-banner.liquid" with banner = homepage_banner %}
```

Comment:
Disappointing to see the React widget go away, but our hands out tied right now because there isn't a great way to load the config dynamically if we can't request it off of githubusercontent.com. I was thinking we could create a route in vets-api that would essentially proxy it to the browser, but that sounds like over-engineering. It's a shame that this homepage banner is still not supported by Drupal.

Reply:
Hmm, yes it seems like the optimal solution would be this banner coming from Drupal 🤔

Reply:
The data _has_ to come from Drupal, regardless of what we do with it on the FE. We can't create a good implementation until then.